clients/unshaded/src/main/java/tachyon/client/lineage/options/CreateLineageOptions.java

```diff
@@ -16,36 +16,39 @@
 package tachyon.client.lineage.options;
 
 import tachyon.annotation.PublicApi;
-import tachyon.client.ClientContext;
-import tachyon.conf.TachyonConf;
 
 /**
  * Method option for creating lineage.
  */
 @PublicApi
 public final class CreateLineageOptions {
-  private static class Builder {
-    /**
-     * Creates a new builder for {@link CreateLineageOptions}.
-     *
-     * @param conf a Tachyon configuration
-     */
-    public Builder(TachyonConf conf) {}
-
-    /**
-     * @return builds a new instance of {@link CreateLineageOptions}
-     */
-    public CreateLineageOptions build() {
-      return new CreateLineageOptions(this);
-    }
-  }
+  /** Flag indicating whether or not to overwrite, currently unsupported */
```

Comment:
I originally wanted to have some field in all the options, but looking at it now it seems like a bad idea.